mucommander-core/src/main/java/com/mucommander/ui/viewer/FileFrame.java

```diff
@@ -1,147 +1,146 @@
-        } catch (UserCancelledException e) {
-            // May get a UserCancelledException if the user canceled (refused to confirm the operation after a warning)
-            return;
-        }
-
-        // If not suitable presenter was found for the given file
-        if (filePresenter == null) {
-            showGenericErrorDialog();
-            return;
-        }
-
-        AsyncPanel asyncPanel = new AsyncPanel() {
-
-            @Override
-            public JComponent getTargetComponent() {
-                try {
-                    // Ask the presenter to present the file
-                    filePresenter.open(file);
-                } catch (Exception e) {
-                    LOGGER.debug("Exception caught", e);
-
-                    showGenericErrorDialog();
-
-                    dispose();
-                    return filePresenter == null ? new JPanel() : filePresenter;
-                }
-
-                setJMenuBar(filePresenter.getMenuBar());
-
-                return filePresenter;
-            }
-
-            @Override
-            protected void updateLayout() {
-                // Request focus on the viewer when it is visible
-                FocusRequester.requestFocus(filePresenter);
-            }
-        };
-
-        // Add the AsyncPanel to the content pane
-        JPanel contentPane = new JPanel(new BorderLayout());
-        contentPane.add(asyncPanel, BorderLayout.CENTER);
-        setContentPane(contentPane);
-
-        setSize(WAIT_DIALOG_SIZE);
-        DialogToolkit.centerOnWindow(this, mainFrame);
-
-        setVisible(true);
-    }
-
-    private void showGenericErrorDialog() {
-        InformationDialog.showErrorDialog(mainFrame, getGenericErrorDialogTitle(), getGenericErrorDialogMessage());
-    }
-
-    /**
-     * Returns whether this frame is set to be displayed in full screen mode
-     *
-     * @return true if the frame is set to full screen, false otherwise
-     */
-    public boolean isFullScreen() {
-        return (getExtendedState() & Frame.MAXIMIZED_BOTH) == Frame.MAXIMIZED_BOTH;
-    }
-
-    /**
-     * Sets this file presenter to full screen
-     */
-    public void setFullScreen(boolean on) {
-        int currentExtendedState = getExtendedState();
-        setExtendedState(
-                on ? currentExtendedState | Frame.MAXIMIZED_BOTH : currentExtendedState & ~Frame.MAXIMIZED_BOTH);
-    }
-
-    public void setDefaultBounds() {
-        DialogToolkit.fitToScreen(this);
-        DialogToolkit.fitToMinDimension(this, getMinimumSize());
-
-        DialogToolkit.centerOnWindow(this, mainFrame);
-    }
-
-    //////////////////////
-    // Abstract methods //
-    //////////////////////
-
-    protected abstract String getGenericErrorDialogTitle();
-
-    protected abstract String getGenericErrorDialogMessage();
-
-    protected abstract FilePresenter createFilePresenter(AbstractFile file) throws UserCancelledException;
-}
+package com.mucommander.ui.viewer;
```

Comment:
this probably happened after rebase (re-formatting of a whole file)

Reply:
ack